demos/package.json

```diff
@@ -15,11 +15,11 @@
     "@shikijs/core": "1.10.3",
     "d3": "^7.9.0",
     "fast-glob": "^3.3.2",
-    "highlight.js": "^11.10.0",
```

Comment:
I guess those version bumps were necessary because pnpm won't load the correct dependencies? Seems like only minors/patches were updated - however we had a bad time in the past with packages only upgrading a minor but still breaking something. 😃

Do we have a fast way to make sure those package upgrades are not breaking anything?

Reply:
I think I just bumped these while regenerating the lockfile (moving from package-lock to pnpm-lock).

I mean the tests ran against all of these and it seems to be working. I know our testing isn't super comprehensive, but we should trust them a bit haha

Reply:
Ah, here was the issue: https://github.com/ueberdosis/tiptap/actions/runs/12629042554/job/35186222134?pr=5978#step:8:1377

Because the type is re-exported, it needs to be present in node_modules (as opposed to the root workspace). This is actually valid though, it is a hard-dep on the current types exported.